numpy/core/setup_common.py

```diff
@@ -216,6 +216,20 @@ def check_long_double_representation(cmd):
         except (AttributeError, ValueError):
             pass
 
+    # Disable multi-file interprocedural optimization in the Intel compiler on Linux
+    # which generates intermediary object files and prevents checking the
+    # float representation.
+    elif sys.platform != "win32" and cmd.compiler.compiler_type.startswith('intel') \
```

Comment:
PEP8, use parenthesis instead of line continuations.
```
elif (sys.platform != "win32" and cmd.compiler.compiler_type.startswith('intel')
           and '-ipo' in cmd.compiler.cc_exe):
```
Note the exception on vertical alignment for if statements.